You seem to have a linter failure:

```
/home/runner/work/shaka-player/shaka-player/test/util/config_utils_unit.js
  25:1  error  This line has a length of 83. Maximum allowed is 80  @stylistic/max-len
```

--- a/test/util/config_utils_unit.js
+++ b/test/util/config_utils_unit.js
@@ -22,8 +22,10 @@ describe('ConfigUtils', () => {
     expect(valid).toBe(false);
     expect(/** @type {!Object} */(destination.drm.advanced)['testPolluted'])
         .toBe(undefined);
-    expect(Object.getPrototypeOf(/** @type {!Object} */(destination.drm.advanced)))
-        .toBe(Object.getPrototypeOf({}));
+    expect(
+        Object.getPrototypeOf(/** @type {!Object} */(destination.drm.advanced)))
+        .toBe(
+            Object.getPrototypeOf({}));
     expect(destination.drm.advanced['com.widevine.alpha'].videoRobustness)
         .toEqual(['SW_SECURE_DECODE']);
   });

